docs/tutorial/index.rst

```diff
@@ -29,6 +29,7 @@ journey through the product.
    initial_setup
    add_osds
    update
+   metrics
```

Comment:
Since the instructions assume a lot on the part of the user (Ceph cluster, Prometheus) this addition is more of a how-to. Please move it to the how-to section.